src/components/plots/piePlot.ts

```diff
@@ -12,9 +18,18 @@ export module Plots {
    *   "outer-radius" - Accessor determining the distance from the center to the outer edge of the sector
    *   "value" - Accessor to extract the value determining the proportion of each slice to the total
    */
-  export class Pie extends Plot {
+  export class Pie<D> extends Plot {
 
     private _colorScale: Scales.Color;
+    private _innerRadius: AccessorScaleBinding<D, number>;
+    private _innerRadiusExtents: any[];
+    private _innerRadiusExtentProvider: Scales.ExtentProvider<any>;
+    private _outerRadius: AccessorScaleBinding<D, number>;
+    private _outerRadiusExtents: any[];
+    private _outerRadiusExtentProvider: Scales.ExtentProvider<any>;
+    private _sectorValue: AccessorScaleBinding<D, number>;
+    private _sectorValueExtents: any[];
+    private _sectorValueExtentProvider: Scales.ExtentProvider<any>;
```

Comment:
It might be better to use some sort of map to the extents, rather than having tons of new variables.